setup.py

```diff
@@ -111,8 +111,8 @@ def install_dependencies(self):
                                '--init', '--recursive'])
 
     def build_extension(self, ext):
-        extdir = os.path.abspath(os.path.dirname(
-            self.get_ext_fullpath(ext.name)))
+        extdir = os.path.abspath(os.path.join(os.path.dirname(
+            self.get_ext_fullpath(ext.name)), 'gtda', 'externals', 'modules'))
```

Comment:
As a side comment, with Python 3.6, this can also be written as
```py
from pathlib import Path

extdir = (Path(self.get_ext_fullpath(ext.name)).base / 'gtda' / 'externals'/  'modules').resolve()

cmake_args = [f'-DCMAKE_LIBRARY_OUTPUT_DIRECTORY={extdir}',
...
```
(not asking to change)

Reply:
It does look much nicer though... :D

Reply:
I wasn't aware of `pathlib`, seems like a modern replacement for `os` and `global` modules.
For anyone interested into a comparison, if found a detailed [article](https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/)

Reply:
Let's merge this as is and employ @rth's suggestion in a future PR.